WeakAurasOptions/OptionsFrames/TextEditor.lua

```diff
@@ -17,44 +17,76 @@ local textEditor
 local valueFromPath = WeakAuras.ValueFromPath;
 local valueToPath = WeakAuras.ValueToPath;
 
-local tableColor = "|c00ff3333"
-local arithmeticColor = "|c00ff3333"
-local relationColor = "|c00ff3333"
-local logicColor = "|c004444ff"
-
-local colorScheme = {
-  [IndentationLib.tokens.TOKEN_SPECIAL] = "|c00ff3333",
-  [IndentationLib.tokens.TOKEN_KEYWORD] = "|c004444ff",
-  [IndentationLib.tokens.TOKEN_COMMENT_SHORT] = "|c0000aa00",
-  [IndentationLib.tokens.TOKEN_COMMENT_LONG] = "|c0000aa00",
-  [IndentationLib.tokens.TOKEN_NUMBER] = "|c00ff9900",
-  [IndentationLib.tokens.TOKEN_STRING] = "|c00999999",
-  -- ellipsis, curly braces, table acces
-  ["..."] = tableColor,
-  ["{"] = tableColor,
-  ["}"] = tableColor,
-  ["["] = tableColor,
-  ["]"] = tableColor,
-  -- arithmetic operators
-  ["+"] = arithmeticColor,
-  ["-"] = arithmeticColor,
-  ["/"] = arithmeticColor,
-  ["*"] = arithmeticColor,
-  [".."] = arithmeticColor,
-  -- relational operators
-  ["=="] = relationColor,
-  ["<"] = relationColor,
-  ["<="] = relationColor,
-  [">"] = relationColor,
-  [">="] = relationColor,
-  ["~="] = relationColor,
-  -- logical operators
-  ["and"] = logicColor,
-  ["or"] = logicColor,
-  ["not"] = logicColor,
-  -- misc
-  [0] = "|r",
-}
+if not WeakAurasSaved["editor_themes"] then
+  WeakAurasSaved["editor_themes"] = {
```

Comment:
I did that initially so maybe ppl could make their own themes in the future something which I think is unnecesarry now